tests/vcd_role.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from pyvcloud.vcd.org import Org
+from pyvcloud.vcd.test import TestCase
+
+
+class TestRole(TestCase):
+    def test_list_role(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, is_admin=True, resource=logged_in_org)
```

Comment:
does it require is_admin to be True? or will it depend on the config values of the test?

Reply:
It depends on the org value in config.yml file. is_admin here is unnecessary. So I am removing it.